cneuromax/projects/friends_language_encoder/peftmodule.py

```diff
@@ -21,21 +20,25 @@ class PEFTLitModule(BaseLitModule, ABC):
 
     def __init__(
         self: "PEFTLitModule",
-        config: BaseLitModuleConfig,
-        nnmodule: nn.Module,
-        optimizer: partial[Optimizer],
-        scheduler: partial[LRScheduler],
+        config: "BaseLitModuleConfig",
+        peft_config: "PeftConfig",
+        *args: Any,  # noqa: ANN401, ARG002
+        **kwargs: Any,  # noqa: ANN401, ARG002
     ) -> None:
         """."""
-        super().__init__(config, nnmodule, optimizer, scheduler)
+        super().__init__(
+            config,
+            peft_config,
```

Comment:
this is wrong, `BaseLitModule`'s constructor does not admita  `peft_config` argument